Lab_works/Lab_works/ViewController.swift

```diff
@@ -1,50 +1,20 @@
-//
-//  ViewController.swift
-//  Lab_works
-//
-//  Created by Olha Polishchuk on 16.10.18.
-//  Copyright © 2018 Olga. All rights reserved.
-//
-
 import UIKit
 import Alamofire
 import SwiftyJSON
 
 class ViewController: UIViewController {
-
-    var articlesData: [ArticleModel] = []
-    
+    let controllerApiManager: ApiManager = ApiManager()
     @IBAction func pressMeButton(_ sender: UIButton) {
-        fetchPhotosData()
+        let ArticleManager = ApiManager()
+        ArticleManager.fetchPhotosData()
```

Comment:
Correct approach is passing callback to `fetchPhotosData`, so `ViewController` will receive result once things are loaded.
Pay attention to following things:
1) while things should be loaded in background thread, result should be rendered on UI thread
2) make sure that your `ApiManager` will not retain an instance of callback. Otherwise if callback is received and view controller is already dead, you will get an error